src/main/scala/edu/ie3/mobsim/model/TripSimulation.scala

```diff
@@ -1017,17 +974,14 @@ object TripSimulation extends LazyLogging {
   def keepOriginalTrip(
       ev: ElectricVehicle,
       plannedStoredEnergyEndOfTrip: ComparableQuantity[Energy],
-      plannedDestinationPoiType: PoiTypeDictionary.Value,
-      plannedDestinationCategoricalLocation: CategoricalLocationDictionary.Value,
       plannedDestinationPoi: PointOfInterest,
       plannedParkingTimeStart: ZonedDateTime,
       plannedDepartureTime: ZonedDateTime
   ): ElectricVehicle = {
 
     /* Because there is no stop at a charging hub, no trip values need to be saved */
-    ev.setFinalDestinationPoiType(None)
-    ev.setFinalDestinationPoi(None)
-    ev.setRemainingDistanceAfterChargingHub(None)
+    var updatedEv: ElectricVehicle = ev.setFinalDestinationPoi(None)
```

Comment:
You can chain these like `ev.setFinalDestinationPoi(None).setRemainingDistanceAfterChargingHub(None)`. This way, you don't need vars